src/certificates/messages.js

```diff
@@ -0,0 +1,46 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  headingTitle: {
+    id: 'course-authoring.certificates.heading.title',
+    defaultMessage: 'Certificates',
+  },
+  headingSubtitle: {
+    id: 'course-authoring.certificates.heading.subtitle',
+    defaultMessage: 'Settings',
+  },
+  headingActionsPreview: {
+    id: 'course-authoring.certificates.heading.action.button.preview',
+    defaultMessage: 'Preview certificate',
+  },
+  headingActionsPreviewAlt: {
+    id: 'course-authoring.certificates.heading.action.button.preview.alt',
+    defaultMessage: 'Preview certificate',
+  },
+  headingActionsDeactivate: {
+    id: 'course-authoring.certificates.heading.action.button.deactivate',
+    defaultMessage: 'Deactivate',
+  },
+  headingActionsDeactivateAlt: {
+    id: 'course-authoring.certificates.heading.action.button.deactivate.alt',
+    defaultMessage: 'Deactivate',
+  },
+  noCertificatesText: {
+    id: 'course-authoring.certificates.nocertificate.text',
+    defaultMessage: 'You have not created any certificates yet.',
+  },
+  setupCertificateBtn: {
+    id: 'course-authoring.certificates.setup.certificate.button',
+    defaultMessage: 'Set up your certificate',
```

Comment:
[important] This button's design has the text "Add your first certificate" should we use it here?
<img width="444" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/3e076919-3f3f-4bec-abc2-f17d23dd0406">

Reply:
Had no design for this page, took text from legacy. Ill change this!